File: Problem Set 1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = 0;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0 || height > 8);


    for (int i = 0; i < height; i++)
    {
        for (int j = height - 1; j > i; j--)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int l = 0; l <= i; l++)
        {
            printf("#");
        }
        /*for (int m = height - 1; m > i; m--)
        {
            printf(" ");
        }*/
        printf("\n");
    }

}

File: Problem Set 6/mario.py
from cs50 import get_int

height = 0

while height <= 0 or height > 8:
    height = get_int("Height: ")


spaces = height - 1
hashes = 1

# Prints row after row

for row in range(height):
    print(' ' * spaces, end="")
    print("#" * hashes, end="")

    # This is the middle of the row
    print("  ", end="")
    print("#" * hashes)
    
    # Increment/decrement spaces and hashes before moving to the following row
    spaces -= 1
    hashes += 1